WORKSPACE

```diff
@@ -12,6 +13,56 @@ load("//:setup.bzl", "kythe_rule_repositories")
 
 kythe_rule_repositories()
 
+# Tweag Nix Support
+######################
+
+# Replace with http_archive: https://github.com/tweag/rules_nixpkgs/#setup
+_commit = "c061458ec4c0e2bfc373b3aa7c804708ebee5474"
+http_archive(
+    name = "io_tweag_rules_nixpkgs",
+    strip_prefix = "rules_nixpkgs-{}".format(_commit),
+    urls = ["https://github.com/tweag/rules_nixpkgs/archive/{}.tar.gz".format(_commit)],
+    sha256 = "2e39a8d0b373b0427c85bc20f144fcf1f4b0f9d1504d0d0455e00192c0cb2c27",
+)
+
+load("@io_tweag_rules_nixpkgs//nixpkgs:repositories.bzl", "rules_nixpkgs_dependencies")
+rules_nixpkgs_dependencies()
+
+# Define nixpkgs version using the flakes lock file.
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl", "nixpkgs_local_repository")
+nixpkgs_local_repository(
+    name = "nixpkgs",
+    nix_flake_lock_file = "//:flake.lock",
+    nix_file_deps = ["//:flake.lock"],
+)
+
+# Configure the C++ toolchain
+load("@io_tweag_rules_nixpkgs//nixpkgs:nixpkgs.bzl", "nixpkgs_cc_configure")
+nixpkgs_cc_configure(
+    name = "nixpkgs_config_cc",
+    repository = "@nixpkgs",
+    attribute_path = "clang_14",
+    cc_lang = "c++",
+    cc_std = "c++11",
```

Comment:
This needs to be at least C++17 and will presumably be C++20 at some point in the near future (I know Google was actively move that direction when I left).

Reply:
c++11 was an attempt to resolve an issue that eventually got traced to a nixos bug. Could probably be removed.

Reply:
Reverted.